templates/travis/.travis/release.py.j2

```diff
@@ -53,7 +60,7 @@ def validate_redmine_data(redmine_query_url, redmine_issues):
         error_messages.append(f"Verify at {redmine_query_url}")
         raise RuntimeError("\n".join(error_messages))
 
-    return f"{REDMINE_URL}/versions/{milestone_id}.json"
+    return commit_message
```

Comment:
For readability, I suggest to settle on one name and not use `commit_message` and `milestone_url` for the same data.